modules/intro/pages/index.adoc

```diff
@@ -32,15 +32,16 @@ TigerGraph Cloud handles all the complexity of deploying and managing your deplo
 For more information, see xref:cloud:start:overview.adoc[].
 
 == Versioning
-TigerGraph uses https://semver.org/[semantic versioning]. Given a version number `MAJOR.MINOR.PATCH`:
+TigerGraph uses https://semver.org/[semantic versioning]. Versions are numbered in the format `MAJOR.MINOR.PATCH`:
 
-* `MAJOR` version increases when there are incompatible API changes to the previous major version
-* `MINOR` version increases when the added functionality in a backwards compatible manner
-* `PATCH` version increases when there are backwards compatible bug fixes.
+* `MAJOR` version releases indicate API changes to the previous major version resulting in incompatibility
+* `MINOR` version releases indicate added functionality in a backwards compatible manner
```

Comment:
@a-m-thomas Victor actually pointed out before that minor releases also might not be fully backward compatible anymore.